### DISCUSSION TITLE
No response from the system when adding a non-existing project for indexing

### DISCUSSION THREAD

**Discussion by unordered-set** [subquery/subql#1068] [Frontier TestNet - Bug Reporting]
1) open admin http://x.x.x.x:8000/projects
2) Add a project "QmSzPQ9f4U1GERvN1AxJ28xq9B5s4M72CPvotmmv1N2bN7d"
3) nothing happens (expected to have an error message)

**Comment by mzxyz** [comment]
Thanks for reporting this issue, we will fix it soon
